data-prepper-plugins/otlp-source/README.md

```diff
@@ -60,6 +60,34 @@ source:
       max_delay: 5s # defaults to 2s
 ```
 
+### Authentication Configurations
+
+By default, the otlp input is unauthenticated.
+
+The following is an example of how to run the server with HTTP Basic authentication:
+
+```yaml
+source:
+  otlp:
+    authentication:
+      http_basic:
+        username: my-user
+        password: my_s3cr3t
+```
+
```

Comment:
Can you add an example with mTLS?

Reply:
@KarstenSchnitter can you please explain what's needed in the example. I haven't dived deep into the Data prepper and OTel Source auth yet. Is this a feature request or just an example?